frontend/src/plugins/auth.js

```diff
@@ -10,11 +10,20 @@ axios.interceptors.response.use(null, (error) => {
   return Promise.reject(error)
 })
 
-function getJWTPayloadFromCookie() {
+function getJWTPayload() {
+  // use cookie (preferred)
   const jwtHeaderAndPayload = Cookies.get(headerAndPayloadCookieName())
-  if (!jwtHeaderAndPayload) return ''
+  if (jwtHeaderAndPayload) {
+    return jwtHeaderAndPayload.split('.')[1]
+  }
+
+  // use token from response body (fallback)
+  const token = store.state.auth.token
+  if (token) {
+    return token.split('.')[1]
+  }
```

Comment:
I don't really like that we will ship this in production, but I see no quick and easy way around it right now.